### ISSUE TITLE
Making Parallax Banner 100vh and overflowing content.

### ISSUE THREAD

**Issue by hamideha** [RRutsche/react-parallax#86]
I am attempting to make a Parallax banner with a transparent Navbar. I cannot figure out how to make the Parallax banner cover 100% of the screen (100vh).

Also, since I am using a Bootstrap Navbar that can collapse and expand on smaller screens, when the Navbar dropdown is expanded, the content of the Parallax overflows the banner.

![image](https://user-images.githubusercontent.com/54807699/103448729-9631f480-4c6b-11eb-898d-ea65f832b903.png)

I've attached a Codesandbox below that shows the code that I'm using. And the overflow issue is demonstrated in the screenshot attached and it can also be reproduced by reducing the screen size vertically and scrolling down. 

https://codesandbox.io/s/parallax-navbar-example-ipxge?file=/src/App.js

Any help is appreciated!

**Comment by rrutsche** [comment]
Hey, thanks for the codesandbox. I will have a look what's going wrong there.

**Comment by rrutsche** [comment]
I know I'm late (sorry) but I just had a look at your example. I also saw some strange behaviour in the codesandbox but when I clicked on "Open in new window" it worked just fine. I forked and adjusted your example a bit so you don't have to use the background component: https://codesandbox.io/s/parallax-navbar-example-forked-njbl2?file=/src/App.js. 
The error that is shown in the Codesandbox console only seems to appear there and not in the newly opened window.
